utils/json.go

```diff
@@ -214,7 +213,7 @@ func isZeroValue(value interface{}) bool {
 	case int, int32, int64, float32, float64:
 		return v == 0
 	case bool:
-		return v == false
+		return !v
```

Comment:
Should we test whether `value` is `nil` at the entry of this function, in case user has specified `a = null` and the API doesn't return `a`.

Reply:
Good point! I'll add it.